warehouse/query-core/src/test/resources/datawave/query/QueryLogicFactory.xml

```diff
@@ -231,6 +231,18 @@
                 <!-- JEXL is the default Jexl query syntax -->
             </map>
         </property>
+
+        <property name="tableHints">
```

Comment:
If this is actually to be used anywhere, these changes will need to be incorporated into the spring-boot form of generating the QueryLogicFactory.  See the microservices/starters/query/src/main/resources/QueryLogicFactory.xml.